packages/svelte/tests/validator/samples/a11y-missing-attribute-href-aria-disabled/input.svelte

```diff
@@ -0,0 +1 @@
+<a aria-disabled="true" tabindex="-1" aria-label="Back">←</a>
```

Comment:
I'd simplify this a bit to not use aria-label and tabindex (based on https://www.scottohara.me/blog/2021/05/28/disabled-links.html)

```suggestion
<a role="link" aria-disabled="true">Back</a>
```

Reply:
Definitely...I wrongly just copied their example

Reply:
Uh just realized that this will still get a warning for redundant role...should we silence that too @geoffrich ?

Reply:
Yeah, if they have an `<a>` without an href, then `role="link"` is allowed. [a without href](https://w3c.github.io/html-aria/#el-a-no-href) does not have the link role

Reply:
The it's an actual bug...fixing it now